services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+import datetime
+
+from db.models import Ticket, Order, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str, date:
+        datetime = None
```

Comment:
Place each argument on a new line, with proper indentation and formatting.
```suggestion
        tickets: list[dict],
        username: str,
        date: datetime = None
```

Reply:
just mechanical mistake 👍